src/iop/exposure.c

```diff
@@ -875,6 +946,13 @@ void gui_init(struct dt_iop_module_t *self)
 
   GtkWidget *vbox_manual = GTK_WIDGET(gtk_box_new(GTK_ORIENTATION_VERTICAL, DT_BAUHAUS_SPACE));
 
+  g->compensate_exposure_bias = gtk_check_button_new_with_label(g_strdup_printf(_("compensate camera exposure bias (%.1f EV)"),
```

Comment:
Maybe just "compensate camera exposure" for the label to keep is short for small panels.

The long tooltip makes all this clean anyway.

Reply:
done